docs/Reference/APIs/ContractExtensionAPIs.md

```diff
@@ -64,58 +81,87 @@ The api for approving the contract extension on the recipient node. This can be
 * `status`: `bool` indicating if the operation was success or failure
 * `txArgs`: standard transaction arguments with `privateFor` info of all nodes involved in contract extension
 
-#### Examples
+!!!examples
+
+    === "JSON RPC"
+
+        Command:
 
-=== "JSON RPC"
+        ```js
+        curl -X POST http://127.0.0.1:22000 --data '{"jsonrpc":"2.0","method":"quorumExtension_approveExtension", "params":["0xb1c57951a2f3006910115eadf0f167890e99b9cb", true, {"from": "0xed9d02e382b34818e88b88a309c7fe71e65f419d", "privateFor":["QfeDAys9MPDs2XHExtc84jKGHxZg/aj52DTh0vtA3Xc="]}], "id":10}' --header "Content-Type: application/json"
+        ```
 
-    ```jshelllanguage
-    // Request
-    curl -X POST http://127.0.0.1:22000 --data '{"jsonrpc":"2.0","method":"quorumExtension_approveExtension", "params":["0xb1c57951a2f3006910115eadf0f167890e99b9cb", true, {"from": "0xed9d02e382b34818e88b88a309c7fe71e65f419d", "privateFor":["QfeDAys9MPDs2XHExtc84jKGHxZg/aj52DTh0vtA3Xc="]}], "id":10}' --header "Content-Type: application/json"
+        Response:
 
-    // Response
-    {"jsonrpc":"2.0","id":10,"result":"0x8d34a594b286087f45029daad2d5a8fd42f70abb0ae2492429a256a2ba4cb0dd"}
+        ```json
+        {"jsonrpc":"2.0","id":10,"result":"0x8d34a594b286087f45029daad2d5a8fd42f70abb0ae2492429a256a2ba4cb0dd"}
+        ```
+
+    === "geth console"
+
+        Command:
+
+        ```javascript
+        quorumExtension.approveExtension("0x1349f3e1b8d71effb47b840594ff27da7e603d17", true ,{from: "0x0fbdc686b912d7722dc86510934589e0aaf3b55a", privateFor:["BULeR8JyUWhiuuCMU/HLA0Q5pzkYT+cHII3ZKBey3Bo="]})
+        ```
+
+        Response:
+
+        ```json
+        "0x9e0101dd215281b33989b3ae093147e9009353bb63f531490409a628c6e87310"
+        ```
+
+    If the contract is already under the process of extension, api call to extend it again will fail.
+
+    Command:
+
+    ```javascript
+    quorumExtension.extendContract("0x1932c48b2bf8102ba33b4a6b545c32236e342f34", "1iTZde/ndBHvzhcl7V68x44Vx7pl8nwx9LqnM/AfJUg=", "0x0fbdc686b912d7722dc86510934589e0aaf3b55a", {from: eth.accounts[0], privateFor: ["1iTZde/ndBHvzhcl7V68x44Vx7pl8nwx9LqnM/AfJUg="]})
+    ```
+
+    Response:
+
+    ```text
+    Error: contract extension in progress for the given contract address
+        at web3.js:3143:20
+        at web3.js:6347:15
+        at web3.js:5081:36
+        at <anonymous>:1:1
     ```
 
-=== "geth console"
+    The recipient can approve the extension only once. Executing `quorumExtension.approveExtension` once extension process is completed will result in the below error:
+
```

Comment:
this snippet shows now in the json-rpc example as well as the geth console example. It might be better to translate this bit into json-rpc, or leave it out on the json-rpc tab. Same applies to the other quorumExtension examples on this page

Reply:
I reworked the errors